builtin/logical/pki/cert_util.go

```diff
@@ -63,7 +63,7 @@ var (
 	leftWildLabelRegex = regexp.MustCompile(`^(` + allWildRegex + `|` + startWildRegex + `|` + endWildRegex + `|` + middleWildRegex + `)$`)
 
 	// OIDs for X.509 certificate extensions used below.
-	oidExtensionSubjectAltName = issuing.OidExtensionSubjectAltName
+	oidExtensionSubjectAltName = certutil.OidExtensionSubjectAltName
```

Comment:
Could we get rid of this local var as well and just update the backend_test.go and cert_util.go to use certutil.OidExtensionSubjectAltName directly?

Reply:
That makes sense to me - have updated to do that.